commerce-item-selector-web/src/main/java/com/liferay/commerce/item/selector/web/internal/display/context/BaseCommerceItemSelectorViewDisplayContext.java

```diff
@@ -79,14 +80,14 @@ protected String getKeywords() {
 		}
 
 		_keywords = ParamUtil.getString(
-			cpRequestHelper.getRenderRequest(), "keywords");
+			requestHelper.getRenderRequest(), "keywords");
 
 		return _keywords;
 	}
 
-	protected final CPRequestHelper cpRequestHelper;
 	protected final HttpServletRequest httpServletRequest;
 	protected final String itemSelectedEventName;
+	protected final CommerceItemSelectorRequestHelper requestHelper;
```

Comment:
`CommerceCatalogRequestHelper requestHelper` should be `CommerceCatalogRequestHelper commerceCatalogRequestHelper`